src/xdocs/report_issue.xml

```diff
@@ -155,14 +155,45 @@ expected level should be 12. [Indentation]
       </p>
 
       <p>
-        Please provide detailed description of problem and chunks of code and config
-        that you currently have.
+        Please provide a clear and detailed description of the problem and describe what
+        you would want to happen instead. Relevant chunks of code and config that you currently
+        have will definitely facilitate understanding. Any other context or
+        screenshots which elaborate further on the relevance and rational behind
+        the new feature may also be added.
       </p>
 
       <p>
-        Imagine that everything is possible and propose name of the new option and its behaviour.
-        Do not think that your issue is so obvious.
+        A clear and concise description of any alternative or existing
+        solution or feature that you may have considered or seen elsewhere.
       </p>
+
+      <p>
+        Imagine that everything is possible and propose name of the new option and its
+        describe its behaviour.
+        Never think that your issue is obvious.
+      </p>
+
+      <p>
+        Example of Feature Request/Enhancement that we expect:
+      </p>
+      <source>
+Check documentation: https://checkstyle.org/config_whitespace.html#WhitespaceAround
+
+Describe the problem in detail:
+
+/var/tmp $ javac Test.java
+
+/var/tmp $ cat Test.java
+public class Test { // Note
+  private static final int SOMETHING = 1;
+}
+
+Describe how the additional functionality will solve the problem:
```

Comment:
before this line please add request for CLI output (desired output).

Reply:
done